htdocs/core/lib/functions.lib.php

```diff
@@ -14850,3 +14850,42 @@ function recordNotFound($message = '', $printheader = 1, $printfooter = 1, $show
 	}
 	exit(0);
 }
+
+/**
+ * Get socal network for user or societe
+ * @param   int      $userId    user Id
+ * @return  int|array    -1 if KO, array if OK
```

Comment:
array<array{name:string,url:string}>|int<-1,-1> is better/needed.

Reply:
It looks like the type of the User class also needs an update

User::$socialnetworks